frontend/src/components/AudioPlayer/index.js

```diff
@@ -22,116 +22,26 @@ import cloud from "./cloud.png";
 function Audio({ songs, setAudioElement }, props) {
   // const Audio = forwardRef((ref))
   const dispatch = useDispatch();
-  const [isPlaying, setIsPlaying] = useState(false);
   const [currentTime, setCurrentTime] = useState(0);
   const { id } = useParams();
   const song = useSelector((state) => state.songs[id]);
   const songAudio = useSelector((state) => state.audio.song);
+  const playerStatus = useSelector((state) => state.audio.status);
   const [progress, setProgress] = useState(0);
   const [duration, setDuration] = useState(0);
-
-  // console.log(songAudio, "SONGAUDIO");
-  const [audioPlayer, setAudioPlayer] = useState("");
-
-  // const audioRef = useContext(AudioContext);
-  const audioRef = useRef(null);
+  const audioRef = useRef();
   console.log(audioRef.current, "audioREF");
 
   useEffect(() => {
```

Comment:
This effect will detect when the `status` in the audio store is updated, in which case it will call the appropriate methods on the audio player reference. This effect essentially tries to keep the audio player in sync with what's in the audio store.